ios/FluentUI.Demo/FluentUI.Demo/Demos/NavigationControllerDemoController.swift

```diff
@@ -387,7 +387,9 @@ class RootViewController: UIViewController, UITableViewDataSource, UITableViewDe
             guard let cell = tableView.dequeueReusableCell(withIdentifier: BooleanCell.identifier, for: indexPath) as? BooleanCell else {
                 return UITableViewCell()
             }
-            cell.setup(title: "Show rainbow ring on avatar", isOn: showRainbowRingForAvatar)
+            cell.setup(title: "Show rainbow ring on avatar",
+                       isOn: showRainbowRingForAvatar,
+                       isSwitchEnabled: navigationItem.usesLargeTitle)
```

Comment:
Thanks for fixing this case too!
Just spotted another issue (video below): when the large title is used but the avatar is not set to appear:

You can create the constant below and pass in the parameter here.
`let isSwitchEnabled = navigationItem.usesLargeTitle && msfNavigationController?.msfNavigationBar.personaData != nil`

https://user-images.githubusercontent.com/68076145/141203510-22925b66-ff23-4ef4-a4a9-01ea920065cf.mov

Reply:
Done!!

Reply:
Thanks @vickprat!